### DISCUSSION TITLE
CPU usage when Idle

### DISCUSSION THREAD

**Discussion by pendula95** [mylinuxforwork/dotfiles#284] [Q&A]
The whole installation with dotfiles works perfectly and I have no performance issues, Everything is snappy and smooth. The only issue I have is with Idle CPU usage. I am seeing 8-12% CPU utilization on highest end laptop CPU and this makes me unrest. This causes poor battery life and this is one of my main concerns when running a OS on laptop. This is my first arch installation, I moved from minimal Fedora XFCE installation and wanted something even more minimalistic.

I know this probably has nothing to do with dotfiles and additional libraries that the whole script installs but I am curious to see other people experiences. 

How much CPU do you see when idle-ing with only htop/top terminal open?

**Comment by Veicm** [comment]
Normally I also have an idle cpu usage of around 8% but I have a very old two core cpu. But that issue is not causes by the dotfiles it is caused by Hyprland it self. The only way to safe the battery is switching to another desktop environment.

**Reply by Svaerd** [reply]
How about the whole experience? does everything feels snappy and responsive with those old cpus? Is it snappier compared to gnome?

**Reply by Veicm** [reply]
Yes everything feels good I only know about the high cpu usage in idle because I saw it on btop. I used gnome only for a short time but I think hyprland is faster.
